sfusurge/surge_website
# 68 Posthog Analytics Integration

## Describe your changes
This PR adds the default setup for posthog to the website. It also adds to the lib some types for mouse activity and a posthog event wrapper function, but is not used in the code itself until necessary.  

Some extraneous console log commands were removed as they are no longer needed

Also fixed some merge conflicts between main and develop due to rushed changes. Includes removal of Sabina Yelyubayeva from the executives list

## Testing steps 
1. checkout to 68-posthog
2. run `pnpm install` 
3. Add the correct environment variables to your `env.local` file. The credentials can be found on [notion]()
4. launch the application, and open DevTools
5. Note that on page load, and whenever a link or button is clicked, a POST request is sent to the posthog collection server
6. Login to [posthog](https://us.posthog.com) with `sfusurge@gmail.com`. The credentials can be found in the notion
7. Open the Product Analytics and/or Web Analytics tab, and browse through the data. You should hopefully be able to see your own personal activity be recorded and persisted in the dashboard, although it may take some time to register

## Issue ticket number and link
https://github.com/sfusurge/surge_website/issues/68

## Checklist before requesting a review
